src/sentry/static/sentry/app/utils/advancedAnalytics.tsx

```diff
@@ -2,6 +2,10 @@ import * as qs from 'query-string';
 
 import {Organization} from 'app/types';
 import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {
+  eventNameMap as growthEventMap,
+  EventParameters as GrowthEventParameters,
+} from 'app/utils/growthAnalyticsEvents';
```

Comment:
unrelated but thinking we should probably just name these keys this in their exports, so we don't have to alias

Reply:
@EvanPurkhiser good call, I'll take care of that in a future PR